build/tutorials/platform/getting-started-for-users.md

```diff
@@ -0,0 +1,121 @@
+# Getting Started For Users
+
+The purpose of this tutorial is to give a general overview and serve as a starting point for users new to the Avalanche Platform and its ecosystem. General knowledge of cryptocurrency space is assumed, in particular familiarity with Ethereum ecosystem.
+
+We recommend reading this document entirely before going further, as that way you may avoid common pitfalls and problems that new users run into when onboarding onto Avalanche. Also, many of these components and applications depend on each other, so it's best to get a full picture of relationships before diving in. Also, it contains safety warnings and pointers that you need to be familiar with in order not to get scammed and your tokens stolen.
+
+You can find a general overview of the Avalanche Platform [here](https://support.avax.network/en/articles/4135427-avalanche-platform-overview), it will be useful in understanding commonalities and differences between Avalanche and other platforms.
+
+## Funding
+
+As all fees on Avalanche are paid in native token, you will need to have some in order to participate in the Avalanche network. You can get it through [exchanges](https://ecosystem.avax.network/marketplace?tag=exchange). Another way to acquire AVAX is via credit card purchase on [Pangolin](https://app.pangolin.exchange/#/buy). Further ways are explained below.
+
+## Wallet
+
+Cryptocurrency funds are usually accessed through wallets. [Avalanche Wallet](https://wallet.avax.network/) is the official web wallet for the Avalanche platform. You can use it to [create your wallet](https://support.avax.network/en/articles/5315160-creating-a-new-wallet-with-the-avalanche-wallet) for use on the network.
+
+You can also use a [HW Ledger](https://docs.avax.network/build/tutorials/platform/setup-your-ledger-nano-s-with-avalanche) to log into your wallet. Using a HW wallet is the most secure way of accessing the network because your private keys and the passphrase never leave the device.
+
+Once you have your wallet, you may want to send your AVAX [from exchange to your wallet](https://support.avax.network/en/articles/5315157-how-to-send-avax-from-an-exchange-to-the-avalanche-wallet).
+
```

Comment:
"Your wallet is how you send and receive tokens on the Avalanche Network, and can almost be seen like a bank account. Unlike traditional banking, there is no customer support to go to if you lose access to your wallet. It's very important to safely store your private keys in a secure manner, like the password manager of your choice (Enpass, Lastpass, etc). If you do not save your private keys, YOU WILL LOSE ACCESS TO YOUR FUNDS FOREVER."

something along those lines, to stress the importance.

Reply:
added the warning.